test/blackbox-tests/test-cases/dune-init.t/run.t

```diff
@@ -20,6 +20,9 @@ Can init a public library
 Can build the public library
 
   $ (cd _test_lib_dir && touch test_lib.opam && dune build)
+  Warning: No dune-project file has been found. A default one is assumed but
```

Comment:
I'm confused: I thought the plan was for `dune init` do the right thing, i.e. create the `dune-project` file too. Why isn't it created here?

Reply:
`dune init` creates the project once it is called with the `project` argument.

Reply:
Ah, I see. I think we should suggest that and test that our suggestion works.

Reply:
I added a suggestion. `dune init project` is already tested elsewhere, it seems overkill to extract a command out of a printed hint and run it (if that's what you meant)

Reply:
I only meant that we should test that what we suggest works (not that we need to extract the suggestion, I agree that's an overkill).

Reply:
We could also make the name argument to `dune init project` optional. Would make it easier for users to silence the warning.

Reply:
Agreed.